### PR TITLE
Log the untracked exceptions

### PR DESCRIPTION
**Pull request by stefan-caraiman**

### CODE REVIEW

File: argus/recipes/cloud/windows.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def install_cbinit(self, service_type):
         try:
             cbdir = introspection.get_cbinit_dir(self._execute)
             # If the directory already exists, we won't be installing Cb-init.
+            LOG.info("Cloudbase-init is already installed, skipping installation.")
```
Comment (alexcoman) [inline argus/recipes/cloud/windows.py]: Move the log message on the `else` branch.

File: argus/recipes/cloud/windows.py [hunk 2]
```diff
@@ -96,17 +97,21 @@ def _run_installation(self, service_type):
...
             # for whatever reason. In this case, we're falling back
             # to use a scheduled task.
+            LOG.debug("Cannot install, deploying using a scheduled task.")
             self._deploy_using_scheduled_task(installer, service_type)
 
         self._grab_cbinit_installation_log()
 
     def _deploy_using_scheduled_task(self, installer, service_type):
+    	LOG.info("Retrieve the schedule installer for CloudbaseInit.")
+
```
Comment (alexcoman) [inline argus/recipes/cloud/windows.py]: This empty line is not required.

File: argus/recipes/cloud/windows.py [hunk 3]
```diff
@@ -255,6 +260,7 @@ def sysprep(self):
             # Knowing this we have to except this kind of errors.
             # This fixes errors that stops scenarios from getting
             # created on different windows images.
+            LOG.debug("Currently rebooting...")
             pass
```
Comment (alexcoman) [inline argus/recipes/cloud/windows.py]: Remote the `pass` statement.

File: argus/scenarios/base.py [hunk 4]
```diff
@@ -136,6 +136,7 @@ def setUpClass(cls):
             try:
                 os.mkdir(cls.conf.argus.output_directory)
             except OSError:
+                LOG.warning("Could not create the output directory.")
                 pass
```
Comment (alexcoman) [inline argus/scenarios/base.py]: ditto

**Comment by micumatei** [comment]
Nu ar fi utile niste loguri de aici  : 

- https://github.com/cloudbase/cloudbase-init-ci/blob/cf8b2a57b6327a35cefdbe71d5444506e4ce9899/argus/recipes/cloud/windows.py#L84
- https://github.com/cloudbase/cloudbase-init-ci/blob/master/argus/recipes/cloud/windows.py#L250
-

**Pull request merged**